executor/aggfuncs/func_group_concat.go

```diff
@@ -82,9 +85,10 @@ func (e *groupConcat) ResetPartialResult(pr PartialResult) {
 
 func (e *groupConcat) UpdatePartialResult(sctx sessionctx.Context, rowsInGroup []chunk.Row, pr PartialResult) (err error) {
```

Comment:
why change this function?

Reply:
Not chunk size
but the `rowsInGroup` size

Reply:
For stream aggregate, `rowsInGroup` would not be greater than chunk size.

Reply:
yeah, this may cause the problem I mentioned before.